crates/primitives/src/aliases.rs

```diff
@@ -65,6 +65,9 @@ pub type BlockHash = B256;
 /// A block number.
 pub type BlockNumber = u64;
 
+/// A block timestamp.
+pub type BlockTimestamp = u64;
```

Comment:
unsure if we should add this w/o also using it, so maybe a separate pr?

Reply:
I would update https://github.com/alloy-rs/alloy/pull/859 to include it

Same for `TxNonce`